.github/workflows/playwright-cron.yml

```diff
@@ -29,4 +30,9 @@ jobs:
       with:
         name: playwright-report
         path: support-e2e/playwright-report/
-        retention-days: 30
\ No newline at end of file
+        retention-days: 30
+    - name: Notify on Failure
+      if: ${{ failure() }}
+      run: |
+        failedWorkflowRun="https://github.com/guardian/support-frontend/actions/runs/$GITHUB_RUN_ID"
+        curl -X POST -H "Content-Type: application/json" "$GOOGLE_CHAT_WEB_HOOK" -d '{"text": "🚨 The Playwright post deployment tests for support frontend have failed! <users/all>\n\nIf you do not know what this means, please reply and ask to pair with someone who does! \n\n👉 Visit first to check failing tests: <'"$failedWorkflowRun"'|Workflow run> \n👾 Possible related PR: https://github.com/guardian/support-frontend/pull/${{ github.event.number }} \n\n 📖 <https://github.com/guardian/support-frontend/wiki/Post-deployment-test-runbook|Check the runbook for a step by step guide>"}'
```

Comment:
Should we have a new message here? It's not post-deployment

Reply:
They are still post-deployment tests = but good shout, I'll differentiate between smoke and cron.